### ISSUE TITLE
discussion: could there be any correlation between this work and google night sight?

### ISSUE THREAD

**Issue by xXDarkCubeXx** [cchen156/Learning-to-See-in-the-Dark#99]
google pixel 3 and 4's night sight appear to have very good results. is it possible that they got inspired by this paper to build their night sight mode? if no, how could they have done it and even improved it. on a further note, is it possible to deploy this model into a phone and do the calculations locally?

**Comment by HimariO** [comment]
Night sight model this paper proposes, in my opinion, isn't robust enough for production. So the chance that pixel phones using this work to create night sight mode is pretty low. If I'm not mistaking, Google had already published their paper about night sight at SIGGRAPH 2019([Handheld Mobile Photography in Very Low Light](https://arxiv.org/abs/1910.11336)). And there is a direct comparison between SID and google's night sight algorithm in the paper.

**Comment by xXDarkCubeXx** [comment]
Did they patent it or something, why don't we have more phones using their technic to get low light photos?

**Comment by HimariO** [comment]
I'm not sure about the patent, google had patented so many things include technique we use daily like batch normalization for defensive purpose. The reason why you don't see many phone manufacture using a similar frame stacking algorithm to get low-light photos probably due to issue like:

- How to make the algorithm running real-time consistently(with high-resolution image)
- How to fine-tune algorithm for the camera sensors you are targeting for
- How to collect enough data for training ML model
- ... and the list will go on depending on the product required

That said, there are still some software companies that manage to pull this off, like [NeuralCam](https://neural.cam/).
